setup.py

```diff
@@ -358,6 +358,7 @@ def install_distributions(distributions):
         gen_data_files('dist-packages')
     )
 
+data_files += [(os.path.join(kalite.ROOT_DATA_PATH, "database"), [os.path.join("kalite", "database", "data.sqlite")])]
```

Comment:
I think it's kind of this pattern... this from a commented out version though, because it describes a possible pitfall.. it's from kalite/legacy/blah_settings.py

```
# This is the standard method... but doesn't work because we cannot load
# kalite.i18n while loading settings because of its __init__.py
# from pkgutil import get_data
# I18N_DATA_PATH = get_data("kalite.i18n", "data")
```